tests/unit/amo/components/TestFeaturedCollectionCard.js

```diff
@@ -55,6 +55,20 @@ describe(__filename, () => {
     expect(root.find(LandingAddonsCard)).toHaveProp('className', className);
   });
 
+  it("renders a theme class name to LandingAddonsCard if the collection's isTheme property is true", () => {
+    const root = render({ isTheme: true });
+    expect(root.find(LandingAddonsCard)).toHaveClassName(
+      'FeaturedCollection--theme',
+    );
+  });
+
+  it("does not render a theme class name to LandingAddonsCard if the collection's isTheme property is false", () => {
```

Comment:
See above for my comment about the descriptions of these tests. I think you should remove the part about it being a collection's isTheme property.